### PR TITLE
feature #229: publish snapshot to bintray

### PR DESCRIPTION
**Pull request by missedone**
hi @cbeust 
PR for issue #229, please have a look at `scripts/upload-snapshot`, once it's OK, could you tune `new-release` (e.g. add -beta flag as command line argument) to call this script to publish snapshots to bintray

ping @juherr

### CODE REVIEW

File: scripts/upload-snapshot [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# please don't share `~/.bintray.conf` to VCS since it must contain credentials as below:
```
Comment (missedone) [inline scripts/upload-snapshot]: @cbeust 
please take care of this, can prepare one conf before you do some test.

see how you can get the api token of bintray: https://bintray.com/docs/usermanual/interacting/interacting_editingyouruserprofile.html#anchorAPIKEY

**Comment by missedone** [comment]
the bintray repo is: https://bintray.com/testng-team/testng-eclipse/updatesite-beta/view
once upload success, you can verify it by add update site: https://dl.bintray.com/testng-team/testng-eclipse in your eclipse and see if the latest plugins listed.

**Comment by juherr** [comment]
@missedone 
Travis should be able to deploy beta like we are doing it with snapshots: https://github.com/cbeust/testng/blob/master/.travis.yml#L11

In fact, it looks like travis is already injecting some env vars: https://travis-ci.org/cbeust/testng-eclipse/jobs/111253262
```
export artifactory_user=cbeust
export artifactory_password=XXX
```

You should be able to add the ones you want via https://travis-ci.org/cbeust/testng-eclipse/settings

**Comment by missedone** [comment]
travis build file updated

**Comment by juherr** [comment]
:+1: Just have to test it once merged ;)

**Comment by missedone** [comment]
looks good in the travis build: https://travis-ci.org/cbeust/testng-eclipse

**Comment by cbeust** [comment]
LGTM. Do you need anything from me? Looks like it's all working.

**Pull request merged**